spec/index.html

```diff
@@ -1466,164 +1473,233 @@ <h2>Generalizations of RDF Triples, Graphs, and Datasets</h2>
+  <section id="section-classicize-caveat">
+    <h2>Limitations</h2>
 
-    <p>Steps <a href="#unstar-error1">4.1.1</a> and <a href="#unstar-error2">4.2.1</a> of the <a href="#section-unstar-algo">`unstar`</a> algorithm exit with an error.
-      This will occur if the input dataset contains at the same time <a>triple terms</a> and a graph named `rdf:unstarMetadata`.
-      The `unstar` algorithm is therefore not strictly universal as it can not transform this particular kind of datasets.
+    <p>The two transformations above are explicitly not supporting graphs or datasets containing at the same time a [=triple term=]
+      and an [=asserted triple=] (<var>b</var>, `rdf:type`, `rdf:TripleTerm`) where <var>b</var> is a [=blank node=].
+      This means, in particular, that the [=classicize=] transformation is not strictly <em>universal</em>.
     </p>
 
     <p>This limitation should not be an issue in practice.
-      The special graph name `rdf:unstarMapping` is unlikely to be in used in any published dataset,
-      as it was not defined in the RDF namespace prior to this specification.
+      The `rdf:TripleTerm` type is unlikely to be in used in any published graph or dataset,
+      as it was not defined prior to this specification.
       For this reason, using it would actually have been bad practice.
-      As for future datasets, their authors should consider the graph name `rdf:unstarMetadata` to be reserved, in order to prevent interference with the `unstar` algorithm.
+      As for future graphs and datasets, their authors should consider this type to be reserved, in order to prevent interference with the [=classicize=] transformation.
     </p>
 
-    <p>Another consequence of this restriction is that users should be careful when merging dataset in an application that makes use of the `unstar` algorithm.
-      More precisely, merging a [=Full=] <a>RDF dataset</a> (containing at least one <a>triple term</a>)
-      with a [=Classic=] <a>RDF dataset</a> resulting from the application of `unstar` (and therefore containing an `rdf:unstarMetadata` named graph)
-      would result in a "hybrid" dataset that `unstar` can not process.
-      Such application should make sure to apply `unstar` to every dataset priori to merging them.
-      Since `unstar` is idempotent, there is no harm in applying it more than necessary.</p>
+    <p class=note>
+      This is one of the reasons why this transformation introduces a new vocabulary
+      (`rdf:TripleTerm`, `rdf:ttSubject`, `rdf:ttPredicate`, `rdf::ttObject`),
+      rather than repurposing the existing <a data-cite="RDF12-SCHEMA#ch_reificationvocab">reification vocabulary</a>
+      (`rdf:Statement`, `rdf:subject`, `rdf:predicate`, `rdf:object`).
+      Contrarily to `rdf:TripleTerm`, `rdf:Statement` is known to used in widely used datasets (e.g. <a href="https://www.uniprot.org/">Uniprot</a>),
+      so deprecating its usage as "reserved" was not an option.
```

Comment:
```suggestion
      This is one reason why this transformation introduces new vocabulary terms
      (`rdf:TripleTerm`, `rdf:ttSubject`, `rdf:ttPredicate`, `rdf::ttObject`),
      rather than repurposing the existing <a data-cite="RDF12-SCHEMA#ch_reificationvocab">reification vocabulary</a>
      (`rdf:Statement`, `rdf:subject`, `rdf:predicate`, `rdf:object`).
      Unlike `rdf:TripleTerm`, `rdf:Statement` is known to be found in
      widely used datasets (e.g., <a href="https://www.uniprot.org/">Uniprot</a>),
      so reserving its use for the [=classicize=] transformation was not an option.
```